### ISSUE TITLE
Can't load highcharts

### ISSUE THREAD

**Issue by 4lm** [rl-institut/WAM_APP_stemp_abw#9]
Hi @nesnoj and @henhuy,

I tried to load the highcharts  "visualizations1" and "visualizations2" from the context in map.html, which are initialized in the class MapView() in __init__.py in stemp_abw/views/__init__.py

In each case, I get three HTML elements (with the content: "Loading...."), but no charts:

- visualizations1:
```
<div id="hc_0" style="display: inline-blockheight:56%;">Loading....</div>
<div id="hc_1" style="display: inline-blockheight:56%;">Loading....</div>
<div id="hc_2" style="display: inline-blockheight:56%;">Loading....</div>
```
- visualizations2:
```
<div id="hc_3" style="display: inline-blockheight:56%;">Loading....</div>
<div id="hc_4" style="display: inline-blockheight:56%;">Loading....</div>
<div id="hc_5" style="display: inline-blockheight:56%;">Loading....</div>
```
I tried to implement the highcharts in the footer (only for testing purposes) with:
```
{% for v in visualizations1 %}
  {{ v }}
{% endfor %}

{% for v in visualizations2 %}
  {{ v }}
{% endfor %}
```
Do you have any ideas? I slightly recall that some second element has to be loaded (at the bottom of map.html), maybe JS code from context, but I'm not quite sure ...

**Comment by nesnoj** [comment]
I'll have a look tomorrow..

**Comment by henhuy** [comment]
You have to add `{{v.media}}` (containing the JS) below your `foundation.app.js`

**Comment by 4lm** [comment]
Yes, adding {{v.media}} worked! Many thanks!

**Comment by nesnoj** [comment]
Great.
Due to the API update, the result highcharts are currently not loaded.
Therefore I reopen the issue, and I will take care of this later.

Last working WAM version with old API: 62a2b99701cc5418b088bf1b043c0ebbbca40755

**Comment by nesnoj** [comment]
Solved in #17
